### DISCUSSION TITLE
Adding content to readme file

### DISCUSSION THREAD

**Discussion by Ananya88** [TesseractCoding/NeoAlgo#2380] [Ideas]
Hey !!
I am interested in this project as GSSOC '21 participant .
And I would like to to update the Readme.md file by adding :

1.How to contribute in this project. The procedure from forking to making a pull request.
2.About how to add issues and etc.

Please assign me with this issue .Thanks.

**Comment by HarshCasper** [comment]
Hi @Ananya88 

Thanks for this proposal! 

Can you present us a rough wireframe of how the README would look like? You can either mock it up or display us a rough layout. We want to have a beautiful and eye-catching README that can help new contributors quickly onboarded. 

The reason why we are asking for a rough mockup is because we might be able to give you a feedback rather than wasting a lot of your and our time on PR reviews. And if the proposal is not upto the mark, we will let you know here itself so that we don't block you from working on other issues. 

Let me know what works for you. Thanks!

**Reply by Ananya88** [reply]
Sure I'll prepare one .
